listenbrainz_spark/recommendations/candidate_sets.py

```diff
@@ -1,3 +1,18 @@
+# This script is responsible to create candidate sets for all users. The generated candidate sets
```

Comment:
We should make this a docstring, so that editors are able to pick it up.

Reply:
Also, let's define what a  candidate set is, it's not obvious to a newbie (which admittedly I am)